src/psycopmlutils/data_checks/flattened/feature_describer.py

```diff
@@ -95,6 +98,9 @@ def generate_feature_description_df(
     # Convert to dataframe
     feature_description_df = pd.DataFrame(rows)
 
+    # Sort feature_description_df by Predictor df
+    feature_description_df = feature_description_df.sort_values(by="Predictor df")
+
```

Comment:
why sorting?

Reply:
Since the final df is created by concatenating source_dfs in whichever order `pool.map` returns them, the feature_description df has rows in random order. Much easier to read if e.g. all `hba1c` features are right after one another :-)

Reply:
Have added a comment explaining it.

Reply:
I see. You're rarely reading the values though, and the rest of the script doesn't care. Might be worth it to skip the sorting for performance and sort if one needs it for debugging?

Reply:
Well, the `feature_description_df.shape[0] == n_features`. So at most ~500 rows? Super fast! So I think it's OK ;-)

Reply:
Ah of course - and we'd also like the feature description to be readable -- keep it by all means!